FirebaseUI.podspec

```diff
@@ -4,8 +4,8 @@ Pod::Spec.new do |s|
   s.summary      = 'UI binding libraries for Firebase.'
   s.homepage     = 'https://github.com/firebase/FirebaseUI-iOS'
   s.license      = { :type => 'Apache 2.0', :file => 'FirebaseUIFrameworks/LICENSE' }
-  s.author       = 'Firebase'
   s.source       = { :http => 'https://github.com/firebase/FirebaseUI-iOS/releases/download/v0.5.6-rc3/FirebaseUIFrameworks.zip' }
```

Comment:
Let's go for 0.6.0 for parity with Android. Not sure if this gets fixed up in the build system.